tests/unit/etl/TransformerTests.cpp

```diff
@@ -60,21 +60,14 @@ struct ETLTransformerTest : util::prometheus::WithPrometheus, MockBackendTest {
     using TransformerType = etl::impl::
         Transformer<ExtractionDataPipeType, LedgerLoaderType, LedgerPublisherType, AmendmentBlockHandlerType>;
 
-    void
-    SetUp() override
+    ETLTransformerTest()
     {
         state_.isStopping = false;
         state_.writeConflict = false;
         state_.isReadOnly = false;
         state_.isWriting = false;
     }
 
-    void
-    TearDown() override
-    {
-        transformer_.reset();
```

Comment:
Don't need this, as destructor will do the same

Reply:
There was some reason why it was added but other code that depended on it may have been later removed. I recall we had some issues with this before

Reply:
Got it, seems to work fine now, so probably the code has been removed.